src/main/java/flight/reservation/Passenger.java

```diff
@@ -1,15 +1,9 @@
 package flight.reservation;
 
-public class Passenger {
-
-    private final String name;
+import javax.xml.namespace.QName;
 
+public class Passenger extends Person {
```

Comment:
introducing Person as a class, the Passenger class seems a bit "empty". Person class itself also seems empty, is the idea to distinguish passengers and crew? Maybe just have either Person or Customer with all the info and a ScheduledFlight could then have a List<Customer> passengers?